src/battle_tower.c

```diff
@@ -3107,9 +3030,9 @@ static void FillPartnerParty(u16 trainerId)
             }
             CalculateMonStats(&gPlayerParty[i+3]);
```

Comment:
```suggestion
            CalculateMonStats(&gPlayerParty[i + 3]);
```
There's a bunch of style issues across this file, actually (like `++j` in loops). Could be useful to just clean them up as well